.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+// prettier.config.js or .prettierrc.js
```

Comment:
seems like it has to .prettierrc   (no .js) for Idea to like it.    This is also what they have in prettier repo itself.